CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- Updated `smithay-clipboard` to `0.6.0`
```

Comment:
Is this the relevant information from this change, rather than mentioning the issues that were fixed? Unless I have to access smithay-clipboard myself, I don't care if it has been bumped.

Reply:
>Is this the relevant information from this change, rather than mentioning the issues that were fixed? Unless I have to access smithay-clipboard myself, I don't care if it has been bumped.

It's not. We can remove it, however bumping version without changelog seems a bit strange.

Reply:
Why is this version bumped, what does it solve? Can't we say that?

Reply:
it just updated smithay-client-toolkit to 0.12 and that's it. Also it changed API to `unsafe`, but it doesn't affect anything in our impl, since we already marked it as such... So it's not a bug fix, it's just an API correction there.

Reply:
Okay, so you do not expect any changes in functionality at all, the goal is just to have everything up to date? Are there any other dependencies we could update?

Reply:
The main point why I want to update it, is that it'll match winit's smithay-client-toolkit version which should reduce a lot of deps.

I've also updated `win-clipboard` here as well.

Reply:
Then I'd just point out that we updated dependencies in general, that should be all the information people know.

That makes it clear why we did it and doesn't confuse anyone with too much detail.